packages/contracts/contracts/foundry/SphinxUtils.sol

```diff
@@ -276,46 +279,17 @@ contract SphinxUtils is SphinxConstants {
             ISphinxScript(_script).configureSphinx();
         }
 
-        SphinxConfig memory config = ISphinxScript(_script).sphinxFetchConfig();
+        UserSphinxConfig memory config = ISphinxScript(_script).sphinxFetchConfig();
         validate(config);
         return config;
     }
 
-    function assertValidOwners(SphinxConfig memory _config) private pure {
```

Comment:
Since registration now occurs via the website, these checks are not necessary here. Instead, I've implemented the same set of checks in the website mutation where the project is created.